example/cumulus-tf/variables.tf

```diff
@@ -221,6 +221,12 @@ variable "elasticsearch_client_config" {
   default     = {}
 }
 
+variable "elasticsearch_remove_index_alias_conflict" {
+  type = bool
+  default = true
```

Comment:
This is set to true for development deployments (e.g. the /example/cumulus-tf stacks) only.    For users it will be false.